### PR TITLE
Prefix CATTLE_HOST_LABELS= to machine labels

### PR DESCRIPTION
**Pull request by wlan0**
@cjellick @will-chan @sonchang

### CODE REVIEW

**Comment by wlan0** [comment]
Fixes https://github.com/rancherio/rancher/issues/999

File: handlers/bootstrap.go [hunk 1]
```diff
@@ -139,8 +140,11 @@ func buildContainerConfig(containerCmd []string, machine *client.Machine, imgRep
 			continue
 		}
 		labelPair := key + "=" + label
-		envVars = append(envVars, labelPair)
+		labelVars = append(labelVars, labelPair)
 	}
+	labelVarsString := strings.Join(labelVars, "&")
+	labelVarsString = "CATTLE_HOST_LABELS=" + labelVarsString
+	envVars = append(envVars, labelVarsString)
```
Comment (cjellick) [inline handlers/bootstrap.go]: What if machine.Labels is empty? Seems to me that we would not want to add "CATTLE_HOST_LABELS=" (without a value) to the run command.

Comment (wlan0) [inline handlers/bootstrap.go]: It'll just be empty. I tested this scenario. It works just fine.

Comment (cjellick) [inline handlers/bootstrap.go]: Right, should work fine. But _why_ set an empty env var?

Comment (wlan0) [inline handlers/bootstrap.go]: I dont have a reason. I can remove it :)

**Comment by wlan0** [comment]
@cjellick updated the PR based on your comments

**Comment by cjellick** [comment]
Waiting for build to go green (currently queued up) and to see the accompanying validation-test PR.

**Pull request merged**